packages/@aws-cdk/integ-tests/lib/assertions/sdk.ts

```diff
@@ -142,7 +142,7 @@ export class AwsApiCall extends Construct implements IAwsApiCall {
         flattenResponse: Lazy.string({ produce: () => this.flattenResponse }),
         salt: Date.now().toString(),
       },
-      resourceType: `${SDK_RESOURCE_TYPE_PREFIX}${this.name}`,
+      resourceType: `${SDK_RESOURCE_TYPE_PREFIX}${this.name}`.substring(0, 60),
```

Comment:
```suggestion
      resourceType: `${SDK_RESOURCE_TYPE_PREFIX}${this.name}`.substring(0, 59),
```

Add a test to be sure 🙂

Reply:
The `substring` method isn't inclusive for the `indexEnd` so it should be `substring(0, 60)`